tml/org.testeditor.tml.dsl.tests/src/org/testeditor/tml/dsl/validation/TmlVarUsageValidatorTest.xtend

```diff
@@ -54,7 +55,7 @@ class TclVarUsageValidatorTest extends AbstractParserTest {
 		''')
 
 		// when
-		tclValidator.checkVariableUsageWithinAssertionExpressions(testStepContext)
+		tclValidator.checkVariableUsageWithinAssertionExpressions(testStepContext.assertInstanceOf(ComponentTestStepContext))
```

Comment:
The `assertInstanceOf` check should not happen in the `// when` step. You can put it in the `// given` step, though.